canonical-web-and-design/maas-ui
# Add System card and grid layout to machine details

## Done

- Add the card grid and system card to the machine details page.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the React machine details.
- See that the summary has a System card and placeholders for the other cards.

## Fixes

Fixes: canonical-web-and-design/MAAS-squad#2109
